system/bootstrap.php

```diff
@@ -10,8 +10,9 @@
  */
 
 use CodeIgniter\CodeIgniter;
+use CodeIgniter\Config\BaseService;
```

Comment:
Using `BaseService` instead of `Config\Services` deprives users to use their own implementation of `autoloader`.

Reply:
I see that, but i think the framework's Autoloader shouldn't be extended or replaced with custom autoloader.
is there any benefit in using a custom autoloader?

Reply:
As the class is not declared "final", it can be extended to change the behavior of some of its methods. Users may have their own peculiar use case for extending the autoloader, like fitting for their specific needs or doing a workaround. For starters, I have extended Autoloader in one of my projects way back to work around a behavior so other users might be doing the same.

Reply:
Thank you, I revert some changes, I think our framework autoloader is strong enough to not be replaced or extended and should be declare as final class, maybe later we shall do this change.